app/models/spree_klaviyo/newsletter_subscriber_decorator.rb

```diff
@@ -0,0 +1,47 @@
+module SpreeKlaviyo
+  module NewsletterSubscriberDecorator
+    def self.prepended(base)
+      base.store_accessor :private_metadata, :klaviyo_subscribed
```

Comment:
Let's use metafields now as we should move away from using `public_metadata` and `private_metadata`